app/poros/search_results.rb

```diff
@@ -1,10 +1,7 @@
 class SearchResults
-  def initialize()
-    @quote_service = QuoteService.new
-  end
-
   def get_quote
-    quote_obj = @quote_service.daily_quote
+    response = Faraday.get("https://jam-session-ms.herokuapp.com/quote")
+    quote_obj = JSON.parse(response.body, symbolize_names: true)
     @quote = Quote.new(quote_obj)
```

Comment:
Right on, this was a much simpler solution than the very wrong way I was taking it. Great work.